content/reusable-snippets.mdx

```diff
@@ -3,29 +3,115 @@ title: Reusable Snippets
 description: Reusable, custom snippets to keep content in sync
 ---
 
-One of the core principles of software development is DRY (Don't Repeat Yourself). This is a principle that we should apply to our documentation as well. If you find yourself repeating the same content in multiple places, you should consider creating a custom snippet to keep your content in sync.
+One of the core principles of software development is DRY (Don't Repeat
+Yourself). This is a principle that we should apply to our documentation as
+well. If you find yourself repeating the same content in multiple places, you
+should consider creating a custom snippet to keep your content in sync.
 
-## Creating and using a custom snippet
+## Creating a custom snippet
 
-1. Create a new file in the `_snippets` directory.
-2. Construct your snippet content - it can be a component or a chunk of text.
+1. Create a new file in the `snippets` directory.
 
-```md _snippets/my-snippet.mdx
-## My Snippet
+<Note>
+  Any page in the `snippets` directory will be treated as a snippet and will not
+  be rendered into a standalone page. If you want to create a standalone page
+  from the snippet import the snippet into another file and call it as a
+  component.
+</Note>
+
+### Default export
+
+2. Add content to your snippet file that you want to re-use across multiple
+   locations. Optionally, you can add variables that can be filled in via props
+   when you import the snippet.
+
+```md snippets/my-snippet.mdx
+Hello world! This is my content I want to reuse across pages. My keyword of the
+day is {word}.
```

Comment:
I think the whole idea of having step 1 at the top and then starting from 2 isn't the best practice. I would make step 1 some sort of pre-condition, and then either starting with 1 for all of the sections or not use numbering at all

Reply:
changed to a precondition + step 1, 2 etc.